package.json

```diff
@@ -48,6 +48,7 @@
   },
   "dependencies": {
     "antd": "^3.16.3",
+    "babel-plugin-import": "^1.11.0",
```

Comment:
This plugin should be required as a dependency by module `packages/api-explorer`.